xline-client/src/clients/lease.rs

```diff
@@ -0,0 +1,44 @@
+// TODO: Remove these when the placeholder is implemented.
+#![allow(dead_code)]
+
+use std::sync::Arc;
+
+use curp::client::Client as CurpClient;
+use tonic::transport::Channel;
+use xline::server::Command;
+
+use crate::AuthService;
+
+/// Client for Lease operations.
+#[derive(Clone, Debug)]
+pub struct LeaseClient {
+    /// Name of the LeaseClient
+    name: String,
+    /// The CURP client
+    curp_client: Arc<CurpClient<Command>>,
+    /// The standalone client
+    standalone_client: xlineapi::LeaseClient<AuthService<Channel>>,
```

Comment:
Why should `LeaseClient` be placed in the `xlineapi` crate while the others, like `CurpClient` or `WatchClient`, are in the `xline-client`? By the way, what does the standalone mean? Can we rename it as lease_client?

Reply:
Not sure what you mean, the `xlineapi::LeaseClient` and `xlineapi::WatchClient` are generated from the protobuf files. And `CurpClient` is defined in the `curp` crate.

The `standalone` is relative to the `curp` client, which means curp client need to communicate with multiple server while the standalone client only communicate with one at a time.

Reply:
I recommend renaming it as `lease_client` and updating the standalone meaning in its comment.

Reply:
done